autogpt/config/ai_config.py

```diff
@@ -29,14 +28,18 @@ class AIConfig:
         ai_role (str): The description of the AI's role.
         ai_goals (list): The list of objectives the AI is supposed to complete.
         api_budget (float): The maximum dollar value for API calls (0.0 means infinite)
+        plugins (list): The list of objectives the AI is supposed to complete.
     """
 
+    SAVE_FILE = str(Path(os.getcwd()) / "ai_settings.yaml")
```

Comment:
Can we guarantee that `cwd` will be the correct dir? Seems like we can use the configured workspace dir.

Reply:
The whole SAVE_FILE approach is weird in my eyes, I am looking into setting the cfg.ai_settings_filepath in main.py so it just has to be set once. it's fixed anyway.